.ci.yaml

```diff
@@ -281,6 +281,54 @@ targets:
       target_file: web_build_all_packages.yaml
       channel: stable
 
+  - name: Linux_web web_platform_tests_shard_1 master
+    bringup: true # New target
+    recipe: packages/packages
+    timeout: 60
+    properties:
+      add_recipes_cq: "true"
+      target_file: web_platform_tests.yaml
+      cores: "32"
```

Comment:
Is this necessary for these targets?

Reply:
These were in the heavy workload in Cirrus, and even then needed to be sharded. I've been mapping heavy workload to 32 cores based on our previous discussion.

If there's a good way to analyze runs after the fact I'm happy to tune this later based on actual usage.